frontend/src/AudioAnnotator/Region.js

```diff
@@ -42,6 +42,7 @@ class Region extends Component<RegionProps> {
 
     const width: number = Math.floor(this.props.timePxRatio * duration);
     const height: number = Math.floor(this.props.freqPxRatio * freqRange) + HEADER_HEIGHT + HEADER_MARGIN;
+    
 
     const styles = {
```

Comment:
The box position is wrong when the header is at the bottom, because styles must be updated when it's the case.

- The `headerPositionTop` must be initialized before the `styles` one
- The `wrapper.top` value must take into account the header position and adjust accordingly: no need to substract header height and margin when header is at the bottom
- The `header.marginBottom` value must take into account the header position: `0` when header is at the bottom
- A property `header.marginTop` must be added, working the other way around the `marginBottom` one.